packages/components/src/inputs/src/RemainingCharacterCount.tsx

```diff
@@ -16,13 +16,13 @@ interface RemainingCharacterCountRenderProps {
 }
 
 export interface RemainingCharacterCountProps extends
-    Omit<ComponentProps<typeof Text>, "children" | "className" | "style">,
     StyledSystemProps,
     Omit<RenderProps<RemainingCharacterCountRenderProps>, "children">
 {
     count: number;
     isDisabled?: boolean;
     isInvalid?: boolean;
+    slot?: string;
```

Comment:
sorry I should have been more clear, you should add an extends called AccessibleSlotProps like you see in RadioField for example.